packages/material-ui-unstyled/src/ModalUnstyled/ModalManager.test.js

```diff
@@ -166,6 +166,62 @@ describe('ModalManager', () => {
       expect(container1.style.paddingRight).to.equal('20px');
       expect(fixedNode.style.paddingRight).to.equal('');
     });
+
+    it('should restore styles correctly if overflow existed before', () => {
+      const modal = {};
+      const container2 = document.createElement('div');
+      container2.style.overflow = 'overlay';
```

Comment:
Let's use a standard value that is supported by our browser matrix. [`overlay` is deprecated](https://developer.mozilla.org/en-US/docs/Web/CSS/overflow#values) and already breaks our Firefox 78 test suite (since it's not supported in Firefox browsers).

Reply:
Yeah I forgot about that one, and wanted to change it to 'scroll' but didn't have time